tests/test_events/test_json.py

```diff
@@ -73,6 +73,22 @@ def test_catalog_can_be_written(self, test_catalog, json_cat_from_disk):
         """
         assert test_catalog == json_cat_from_disk
 
+    def test_single_event(self):
+        """Ensure a single event can be converted to json."""
+        event = obspy.read_events()[0]
+        out = cat_to_json(event)
+        assert isinstance(out, str)
+        cat_again = json_to_cat(out)
+        assert len(cat_again) == 1
+
+    def test_list_of_events(self):
+        """Ensure a list of events can be converted to json."""
+        events = obspy.read_events().events
+        out = cat_to_json(events)
```

Comment:
Should the docstring for `cat_to_json` should be `Union[Catalog, Event, Iterable[Event]]`?